### DISCUSSION TITLE
Node 16

### DISCUSSION THREAD

**Discussion by beeequeue** [vercel/vercel#6251] [Ideas] [answered]
Node 16 is the current LTS version as of April this year, when is support for it getting added?

**Comment by pcjmfranken** [comment]
You might have mixed up Node's `LTS` and `current` version designations. Node 14 is the active LTS, with Node 16 scheduled to take over by the end of October.

Vercel is also dependant on AWS in this. I believe Node 14 has only been properly and fully supported on AWS since the end of last month.

Taken from https://nodejs.org/en/about/releases/: 
Release | Status | Codename | Initial Release | Active LTS Start | Maintenance LTS Start | End-of-life
-- | -- | -- | -- | -- | -- | --
v12 | Maintenance LTS | Erbium | 2019-04-23 | 2019-10-21 | 2020-11-30 | 2022-04-30
v14 | Active LTS | Fermium | 2020-04-21 | 2020-10-27 | 2021-10-19 | 2023-04-30
v15 | Current | &nbsp; | 2020-10-20 | &nbsp; | 2021-04-01 | 2021-06-01
v16 | Current | &nbsp; | 2021-04-20 | 2021-10-26 | 2022-10-18 | 2024-04-30

**Reply by beeequeue** [reply]
I guess I used the wrong term, the correct one would be something like `current LTS`, the latest LTS-eligible version?

Just wanted to know when it will be supported since I couldn't find any info on it! 😄

**Reply by pcjmfranken** [reply]
I believe `active LTS` is the term they would use for that, only slightly confusing 😅

**Comment by osdiab** [comment]
Now that Node 16 is the Active LTS, when can we expect Vercel to announce support for it?

**Reply by andykenward** [reply]
As @pcjmfranken mentioned that Vercel is dependant on AWS supporting it. 
Might be worth keeping an eye out on the below?
- https://github.com/aws/aws-lambda-base-images/issues/14
- https://docs.aws.amazon.com/lambda/latest/dg/lambda-nodejs.html

Anyone else know a better source for AWS Node version support or their roadmap?

**Comment by leerob** [comment, marked as answer]
[Node.js 16 LTS support has been released.](https://twitter.com/vercel_changes/status/1523730208750252033)
